packages/webapp/src/components/AddNewCrop/index.js

```diff
@@ -115,6 +117,11 @@ export default function PureAddNewCrop({
         )}
       />
 
+      <div style={{ marginBottom: '20px', fontSize: '16px' }}>{t('CROP_CATALOGUE.COVER_CROP')}</div>
+      <div style={{ marginBottom: '20px' }}>
+        <RadioGroup hookFormControl={control} name="cover_crop_radio"></RadioGroup>
```

Comment:
I think this should be "can_be_cover_crop" which is the property that's defined in the "cropModel"  instead of cover_crop_radio